CONTRIBUTING.md

```diff
@@ -1,5 +1,5 @@
-# Contributing to TIDES 
-Thank you for contributing to the TIDES Project.  This document defines the roles and process for 
+# Contributing to TIDES
```

Comment:
pre-commit is telling me that headings need to be surrounded by blank lines. Did you get that warning?

Reply:
Yeah - I fixed it all in PR #38 but will fix here too so it is clean.